### ISSUE TITLE
Pattern Update: Bot naming scheme changed

### ISSUE THREAD

**Issue by tadghh** [vikingSec/botbot#1]
Check user profile links, will either be in bio or profile link spot. The follow ratio check should still catch them.
- Check bio along with link section for
 - 'sex'
 - 'meet'
 - '18+'
 - 'odd subdomain'

![image](https://github.com/user-attachments/assets/d0aaac73-0b30-4c16-acb7-e1ce6a2c4607)

Below accounts using format
![image](https://github.com/user-attachments/assets/c168ee4d-c64b-404c-b28c-7778025e13d1)

**Comment by vikingSec** [comment]
Smart! I like that approach.

How do you feel about maybe a blacklist of known-bad domains that can be added to over time? That would be a bit easier than building out a regex to find, say, odd subdomains.

**Comment by tadghh** [comment]
Uh yeah that would be easier, some good actors might catch strays as time goes on. ex:1 perhaps a legitimate place card company comes along and buys the domain, post porn bot. Whats the current list implementation, just client side? If so false positives might be a non issue

ex:1
![image](https://github.com/user-attachments/assets/57b31f09-0a74-4d2f-abbd-155297a7aea0)

**Comment by vikingSec** [comment]
Yeah it would be client-side. super trivial to add to it or remove from it, could also make a PR with additions/deletions to reflect it across other users.

**Comment by tadghh** [comment]
Off topic but is there any specific reason we dont make api calls to block users?

**Comment by vikingSec** [comment]
Mainly price! The API is fairly expensive.

I also wanted folks to be able to cater the plugin for what they wanted. If they have a certain type of user they want to block, I want them to be able to go in the code to allow that. That meant keeping most of the code in the front end, which makes it a bit less attractive to try to do things with a central API key and a back end. I could have required users import their own API keys, but that's a fairly involved process.

Finally, and least importantly, I like the challenge of working around things in this way, and wanted to release something hacky that works. I think I at least achieved the hacky part. 😂
